Scripts/SetIfEmpty/SetIfEmpty.py

```diff
@@ -0,0 +1,20 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+def main(args):
+    demisto.results(getValueToSet(args))
+
+def getValueToSet(args):
```

Comment:
this function should be placed the `main` func

Reply:
Did you mean above the main function?

Reply:
yes